spec/requests/tags_controller_spec.rb

```diff
@@ -164,9 +164,10 @@
 
         it "hides pm tags" do
           get "/tags.json"
-          tags = response.parsed_body["tags"]
-          expect(tags.length).to eq(1)
-          expect(tags[0]["id"]).to eq(topic_tag.name)
+
+          expect(response.parsed_body["tags"]).to match(
+            [include("name" => topic_tag.name, :id => topic_tag.id)],
```

Comment:
```suggestion
            [include(name: topic_tag.name, id: topic_tag.id)],
```

Nit: The syntax is inconsistent here.